Tests/helper.py

```diff
@@ -125,55 +130,68 @@ def assert_image_similar(a, b, epsilon, msg=None):
             + f" average pixel value difference {ave_diff:.4f} > epsilon {epsilon:.4f}"
         )
     except Exception as e:
-        if HAS_UPLOADER:
+        if uploader:
```

Comment:
Same for this `if`.

Reply:
Ok, I've pushed https://github.com/python-pillow/Pillow/pull/7733/commits/f7701e6596c9a2e7c9ed886621f01b3861d70990 for this.